src/RA_CodeNotes.cpp

```diff
@@ -20,35 +22,39 @@ size_t CodeNotes::Load(const std::string& sFile)
     Clear();
 
     SetCurrentDirectory(NativeStr(g_sHomeDir).c_str());
-    FILE* pf = nullptr;
-    if (fopen_s(&pf, sFile.c_str(), "rb") == 0)
+
+    std::ifstream ifile{ sFile, std::ios::binary };
+
+    Document doc;
+    IStreamWrapper isw{ ifile };
+    doc.ParseStream(isw);
+
+    if (!doc.HasParseError())
+
     {
-        Document doc;
-        doc.ParseStream(FileStream(pf));
-        if (!doc.HasParseError())
-        {
-            ASSERT(doc["CodeNotes"].IsArray());
+        ASSERT(doc["CodeNotes"].IsArray());
```

Comment:
You know that assert gets compiled out in release mode right? It might be more appropriate to throw an exception or use a conditional depending on whether it's debug mode or not